backend/src/federation/client/1_0/logging/SendCoinsArgsLogging.view.ts

```diff
@@ -0,0 +1,24 @@
+import { AbstractLoggingView } from '@logging/AbstractLogging.view'
+
+import { SendCoinsArgs } from '@/federation/client/1_0/model/SendCoinsArgs'
+
+export class SendCoinsArgsLoggingView extends AbstractLoggingView {
+  public constructor(private self: SendCoinsArgs) {
+    super()
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public toJSON(): any {
+    return {
+      recipientCommunityUuid: this.self.recipientCommunityUuid,
+      recipientUserIdentifier: this.self.recipientUserIdentifier,
+      creationDate: this.self.creationDate,
+      amount: this.decimalToString(this.self.amount),
+      memoLength: this.self.memo.length,
+      senderCommunityUuid: this.self.senderCommunityUuid,
+      senderUserUuid: this.self.senderUserUuid,
+      senderUserName: this.self.senderUserName.substring(0, 3),
+      senderAlias: this.self.senderAlias?.substring(0, 3),
```

Comment:
it is because of data privacy, debug don't change that. In my opinion personal user data don't belong into the log files.
Only so much to see if the field was filled with the correct data type. From my experience with the gdt server, 3 Character is the perfect balance to see that the correct data is there if you compare with the complete data set but without learning to much about the user, if you don't have access to the complete data set (database access for example).

Reply:
yes i agree